Source/Core/Common/EnumFormatter.h

```diff
@@ -60,22 +60,23 @@ class EnumFormatter
   template <typename FormatContext>
   auto format(const T& e, FormatContext& ctx)
   {
-    const auto value = static_cast<std::underlying_type_t<T>>(e);
+    const auto value_s = static_cast<std::underlying_type_t<T>>(e);      // Possibly signed
+    const auto value_u = static_cast<std::make_unsigned_t<T>>(value_s);  // Always unsigned
+    const bool has_name = value_s >= 0 && value_u < size && m_names[value_u] != nullptr;
```

Comment:
The value_s >= 0 check isn't actually useful for checking the bounds (because of the value_u < size), but I guess it doesn't hurt to check that the index is valid.